projects/browser-extension-testing.md

```diff
@@ -0,0 +1,25 @@
+# Browser extension testing and reliability
```

Comment:
You intend to work on this immediately after shipping https://github.com/sourcegraph/browser-extensions/pull/271, right? (Do you have an ETA on merging that PR?) That sounds right to me. I already moved the rest of "Better config and depoyment for code host UI integrations" to November 2018 on https://docs.sourcegraph.com/dev/roadmap#november-2018 and split out the options page blog post to https://github.com/sourcegraph/about/pull/46.

Reply:
Correct. I just have to pull in access token changes that I made the other day into the new options menu.